translations.yml

```diff
@@ -1090,33 +1097,41 @@
 
 - en: 'Editors:'   # gender-neutral, plural. Precedes list of more than one person who edited the resource. If gender-neutral not in your language, use appropriate alternative, such as plural or 'Edition'.
   fr: 'Rédaction :'
+  ja: '編集者：'
 
 - en: 'Update Editor:'   # gender-neutral, singular. Precedes list of one person who updates the resource. If gender-neutral not in your language, use appropriate alternative, such as plural or 'Update Edition'.
   fr: 'Rédaction des mises à jour:'
+  ja: '更新作業者：'
 
 - en: 'Status:' # followed by the last updated date or
   fr: 'Statut :'
+  ja: 'ステータス：'
```

Comment:
@hiroya-uga When used, one space is added after this UI term (see footer in https://www.w3.org/WAI/tutorials/images/decision-tree/ for example). The space at the end is therefore redundant.